pkg/ui/workspaces/cluster-ui/src/api/databases/tableIndexesApi.ts

```diff
@@ -49,8 +48,8 @@ export const useTableIndexStats = ({
   tableName,
 }: GetTableIndexesRequest) => {
   const makeRequest = dbName && tableName;
-  const { data, isLoading, error, mutate } = useSWR(
-    ["tableIndexes", dbName, tableName],
+  const { data, isLoading, error, mutate } = useSwrWithClusterId(
+    { name: "tableIndexes", dbName, tableName },
```

Comment:
I originally changed it when i had the old hook for adding the clusterId. Mainly I just thought it would be better to have a standardized way that we define our keys (which was using objects instead of arrays). After changing the hook i just didn't change this one back.